### DISCUSSION TITLE
fetch historical data

### DISCUSSION THREAD

**Discussion by soltana80** [ig-python/ig-markets-api-python-library#223] [General]
Hallo,

I tried to fetch historical data of the epic "IX.D.DAX.IFMM.IP" but I get for dates before august only the Exception: Historical price data not found.
The resolution was one minute.

Is it possible that IG does not deliver this data? I did not find anything in the API description.

**Comment by soltana80** [comment]
I searched more and it seems that ig does not deliver more data ...

**Comment by bug-or-feature** [comment]
Read [the FAQs](https://labs.ig.com/faq), there is an item there about amount of historical data available per resolution
